### PR TITLE
Update to validate_aie_pl host code to fix back to back run hangs

### PR DESCRIPTION
**Pull request by jeffli-xilinx**
#### Problem solved by the commit
Fixes aie_pl_test hanging issue after the first run.

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
Discovered during xbutil integration of aie_pl_test to the validate test suite

#### How problem was solved, alternative solutions (if any) and why they were rejected
Updated host code for aie_pl_test from PL controller team

#### Risks (if any) associated the changes in the commit
None 

#### What has been tested and how, request additional testing if necessary
Multiple runs of aie_pl_test

#### Documentation impact (if any)
None

### CODE REVIEW

**Comment by dayeh-xilinx** [comment]
retest this please.

File: tests/validate/aie_pl_test/src/host.cpp [hunk 1]
```diff
@@ -103,23 +103,36 @@ main(int argc, char* argv[])
...
+                                             num_sample);
+    m_pl_ctrl.enqueue_set_and_enqueue_dma_bd("mygraph.first.out[0]", 1,
+                                             num_sample);
+    m_pl_ctrl.enqueue_sync(num_sample);
+    if (num_iter%2 != 0) {
+	m_pl_ctrl.enqueue_set_and_enqueue_dma_bd("mygraph.first.in[0]", 0,
+						 num_sample);
+	m_pl_ctrl.enqueue_set_and_enqueue_dma_bd("mygraph.first.out[0]", 0,
+						 num_sample);
+	m_pl_ctrl.enqueue_sync(num_sample);
```
Comment (dbenusov-xilinx) [inline tests/validate/aie_pl_test/src/host.cpp:130]: These statements should be indented

File: tests/validate/aie_pl_test/src/host.cpp [hunk 2]
```diff
@@ -176,8 +181,18 @@ main(int argc, char* argv[])
...
+    sender_receiver_r1.set_arg(0, num_iter);
+    sender_receiver_r1.set_arg(1, in_bo1);
+    sender_receiver_r1.set_arg(2, out_bo1);
+    sender_receiver_r1.start();
+    std::cout << " start sender-receiver kernel" << std::endl;
+
     controller_r1.wait();
     std::cout << " pl controller wait complete" << std::endl;
+    sender_receiver_r1.wait();
+    std::cout << " sender_receiver wait complete" << std::endl;
```
Comment (dbenusov-xilinx) [inline tests/validate/aie_pl_test/src/host.cpp:195]: Was this what was causing the hang on the second run? The sender_receiver was never collected?

**Review by dbenusov-xilinx — APPROVED** [review]

File: tests/validate/aie_pl_test/src/host.cpp (hunk 2 above)
Comment (jeffli-xilinx) [reply tests/validate/aie_pl_test/src/host.cpp:195]: This change is not needed.  Only the PL controller changes are needed.
Let me revert this change.

File: tests/validate/aie_pl_test/src/host.cpp (hunk 1 above)
Comment (dbenusov-xilinx) [reply tests/validate/aie_pl_test/src/host.cpp:130]: Turns out it was just my git page

**Pull request merged**